src/scripts/medgen_conflicts_removals.py

```diff
@@ -0,0 +1,146 @@
+"""MedGen conflicts: Removals
```

Comment:
## Some things not removed? @matentzn 
@matentzn This still a problem? Could you double check my understanding?

<details><summary>Details</summary>
<p>

[_Nico wrote](https://github.com/monarch-initiative/mondo/commit/cef1663f5aa4ba27eb10be5b2393735f1f02e8cd)_:
> @joeflack4 Why were they these not removed by the remove step?

@matentzn It looks like you corrected a problem [in this commit](https://github.com/monarch-initiative/mondo/commit/cef1663f5aa4ba27eb10be5b2393735f1f02e8cd), but I'm not sure if the problem remains and there is something I need to solve?

If it still needs my fixing, could you double check my understanding and clarify if the following is what's happening?
1. There's a robot template that shows things that should be removed from `mondo-edit.obo`
2. After running pipeline, they're still not removed.

</p>
</details>

Reply:
Sorry, I am creating more work than its worth here. Lets leave it; something in the "conflict removal" step is a bit incomplete, probably because it focusses on combinations of MONDO ids with Xrefs.

In this commit, `UMLS:C1858991` was referenced from a another class (not in the MedGen table) and therefore caused a clash after the fixed xrefs where added in.

Lets not worry about this; it looks mostly good, I would rather we have a decision about how to curate UMLS moving forward!